pagopa/io-app
# chore: [IOBP-123] New list item transaction into IDPay timeline

# ⚠️ This PR depends on #4776 ⚠️

## Short description
This PR adds into IDPay timeline operations the new `ListItemTransaction` component;

## List of changes proposed in this pull request
- `TimelineOperationsListItem.tsx`: Removed the previous ad-hoc component implementation with the new component into DS;
- Adapted business logic to calculate the title, amount, icon and status required in the `ListItemTransaction` component;

## How to test
Into wallet section, open any IDPay initiative and check that the new component is showed correctly into timeline operation list

## Preview

https://github.com/pagopa/io-app/assets/34343582/f53e4c5d-fdec-4af3-ba10-3261ac428d83
